test/cmpd_dtransform.c

```diff
@@ -43,54 +44,58 @@ main(void)
     att_t      *atts        = NULL;
     att_t      *atts_res    = NULL;
 
+    printf("Testing writing compound attributes followed by data w/ transform.\n");
+
+    TESTING("data types are reset properly");
```

Comment:
Does this need to be TESTING_2 if the printf above is a sort of test heading?

Reply:
That's how we enter every test. It's just glommed in with the test output itself since this is a one-off written by a non-HDFG person.